cstore_fdw.c

```diff
@@ -1400,18 +1234,16 @@ CStoreGetOptions(Oid foreignTableId)
 	{
 		blockRowCount = pg_atoi(blockRowCountString, sizeof(int32), 0);
 	}
-
-	/* set default filename if it is not provided */
-	if (filename == NULL)
+	if (loggingString != NULL)
 	{
-		filename = CStoreDefaultFilePath(foreignTableId);
+		logging = strncmp(loggingString, LOGGING_STRING_TRUE, NAMEDATALEN) == 0;
```

Comment:
we could, but it can only have true or false values. parse_bool is more general purpose